internal/service/server/list.go

```diff
@@ -36,6 +37,25 @@ func expandTagListParams(tl []interface{}) ([]*server.InstanceTagParameter, erro
 	return tagList, nil
 }
 
+func expandBlockDevicePartitionListParams(bl []interface{}) ([]*vserver.BlockDevicePartition, error) {
+	blockDevicePartitionList := make([]*vserver.BlockDevicePartition, 0, len(bl))
+
+	for _, v := range bl {
+		blockDevicePartition := new(vserver.BlockDevicePartition)
```

Comment:
(trivial) There's nothing wrong, but I'd like to recommend using conventional/popular forms of pointer type initialization.
```suggestion
		blockDevicePartition := &vserver.BlockDevicePartition{}
```